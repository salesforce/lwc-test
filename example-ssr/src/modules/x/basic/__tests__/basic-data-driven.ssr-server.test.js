import Basic from '../basic';
import { renderAndHashComponent } from '@lwc/jest-ssr-snapshot-utils';
import tests from './test-data';

describe('<x-basic>', () => {
    it.each(tests)('should render on the server (props = $props)', async ({ props }) => {
        const { renderedComponent, snapshotHash } = renderAndHashComponent('x-basic', Basic, props);
        expect(renderedComponent).toMatchSnapshot(snapshotHash);
    });
});
