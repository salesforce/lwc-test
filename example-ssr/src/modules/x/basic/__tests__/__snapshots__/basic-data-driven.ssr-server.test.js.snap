// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<x-basic> should render on the server (props = {"msg": "Hello, Universe!"}): 34628b2c2f600c2775bdf4131a576fcf6bc07e003cef10d0d523ff923d7af4e2 1`] = `"<x-basic><template shadowrootmode="open"><h1>Hello, Universe!</h1></template></x-basic>"`;

exports[`<x-basic> should render on the server (props = {"msg": "Hello, world!"}): ebf35c654b2d8178d7dd129e79ae74bd36c7f5bc3c496babc46f60da73dae93b 1`] = `"<x-basic><template shadowrootmode="open"><h1>Hello, world!</h1></template></x-basic>"`;
